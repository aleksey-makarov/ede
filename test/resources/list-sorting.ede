{
  "single": "Lorem ipsum-Phrase",
  "alpha": ["a", "b", "c", "d", "e"],
  "numeric": [1, 2, 3, 4, 5]
}
---
{{ single | reverse }}
{{ alpha | reverse | show }}
{{ numeric | reverse | show }}
