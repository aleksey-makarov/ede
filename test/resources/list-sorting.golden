esarhP-muspi meroL
["e","d","c","b","a"]
[5,4,3,2,1]
